kazoo/tests/test_queue.py

```diff
@@ -1,43 +1,46 @@
 import uuid
 
-from nose import SkipTest
-from nose.tools import eq_, ok_
+import pytest
 
 from kazoo.testing import KazooTestCase
 from kazoo.tests.util import TRAVIS_ZK_VERSION
 
 
 class KazooQueueTests(KazooTestCase):
-
     def _makeOne(self):
         path = "/" + uuid.uuid4().hex
         return self.client.Queue(path)
 
     def test_queue_validation(self):
```

Comment:
This could be done as a pytest parametrized test... with "input" / "output exception"...